frontend/apps/reader/modules/readertypography.lua

```diff
@@ -86,6 +87,7 @@ local LANGUAGES = {
     { "ru-GB",                 {},   "Hb  ",   _("Russian + English (UK)") },
     { "ru-US",                 {},   "Hb  ",   _("Russian + English (US)") },
     { "ru",               {"rus"},   "Hb  ",   _("Russian") },
+    { "sr",               {"srp"},   "H   ",   _("Serbian") },
```

Comment:
And if you're no bringing a Serbian.pattern, just remove the H from the features string (it's for display in the menu, describing what features are available). Let it as for "ja" and "ko".

Reply:
Sorry about the confusion. Can you please suggest what else I need to add in order to proper add typographic rules for Serbian language?

Reply:
As you added _`duplicate_real_hyphen_on_next_line` for Serbian , you can replace your `H` here with `HB`.